ui/raidboss/data/02-arr/raid/t9.js

```diff
@@ -216,7 +216,7 @@
       id: 'T9 Dragon Locations',
       regex: Regexes.addedCombatantFull({ name: ['Firehorn', 'Iceclaw', 'Thunderwing'] }),
       regexDe: Regexes.addedCombatantFull({ name: ['Feuerhorn', 'Eisklaue', 'Donnerschwinge'] }),
-      regexFr: Regexes.addedCombatantFull({ name: ['corne-de-feu', 'griffe-de-glace ', 'aile-de-foudre'] }),
+      regexFr: Regexes.addedCombatantFull({ name: ['Corne-De-Feu', 'Griffe-De-Glace', 'Aile-De-Foudre'] }),
```

Comment:
Nice, but not necessary, as these are all case insensitive now.

Reply:
the capitalization just haoppend from my script (looks also nicer :P) but originally `griffe-de-glace ` had an extra space at the end